apps/test-suite/tests/Calendar.js

```diff
@@ -313,6 +314,19 @@ export async function test(t) {
         t.expect(typeof eventId).toBe('string');
       });
 
+      if (Platform.OS === 'ios') {
+        t.it('rejects when time zone is invalid', async () => {
```

Comment:
Could we also add a happy test which should pass if the provided `timeZone` is correct? I don't think we have one atm.